build/gulpfile.js

```diff
@@ -4,6 +4,12 @@ var extension  	= require('./package.json');
 var config      = require('./gulp-config.json');
 
 var argv       	= require('yargs').argv;
+
+if (argv.wwwDir)
```

Comment:
sure, could be used there too, it's just i did this for a component using redcore, not for redcore standalone release script, so i didn't think of it